### ISSUE TITLE
node-inspector  can't console in nodev6.2.1 , Enter no reaction

### ISSUE THREAD

**Issue by trojanliu** [node-inspector/node-inspector#951]
node v6.2.1
Node Inspector v0.12.8

i don't know why i can't console in the chrome54.0.2840.98.

Enter no reaction

![image](https://cloud.githubusercontent.com/assets/8016811/20549995/430b3bde-b16c-11e6-8914-992902720cb6.png)

![image](https://cloud.githubusercontent.com/assets/8016811/20510191/bf188948-b0a9-11e6-90c2-dcb8d91fc452.png)

**Comment by Fallenstedt** [comment]
I would give you 100 thumbs up if I could. I've been looking for a work around. Another person has mentioned this before at [#942](https://github.com/node-inspector/node-inspector/pull/942), however their solution did not work for me.

**Comment by trojanliu** [comment]
I find the cause of the problem

Node - node in the inspector - inspector/front - end/platform/DOMExtension. Js  IsEnterKey method, to enter the event judge USES the KeyboardEvent. KeyIdentifier.

but chrome 54.0 has remove  "KeyboardEvent. KeyIdentifier"
![aaaa](https://cloud.githubusercontent.com/assets/8016811/20552576/69ff624a-b187-11e6-87a6-76f49f2ea3f4.jpg)

The solution is change the DOMExtension/isEnterKey 's function: 

vi /usr/local/lib/node_modules/node-inspector/front-end/platform/DOMExtension.js
isEnterKey

function isEnterKey(event) {
    //suit for event.keyIdentifier
    return (event.keyCode !== 229 && event.keyIdentifier === "Enter") ||  event.keyCode === 13;
}

**Comment by bstro** [comment]
Seems like this issue should still be open?

**Comment by zxaos** [comment]
Might have been closed because dupe of #903?

**Comment by TrevorSayre** [comment]
@trojanliu That will fix the Enter key, but not the other keys.

Since it's caused by Chrome deprecating [KeyboardEvent.keyIdentifier](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyIdentifier), the workaround would be to add `keyIdentifier` back to the `KeyboardEvent` prototype.

I also noticed that the `KeyboardEvent.key` string values are different from those from `KeyboardEvent.keyIdentifier` so I show below how to handle those differences if needed.

```javascript
Object.defineProperty(KeyboardEvent.prototype, 'keyIdentifier', {
    get: function() {
        switch (this.key) {
            case "ArrowDown":
                return "Down";
            case "ArrowLeft":
                return "Left";
            case "ArrowRight":
                return "Right";
            case "ArrowUp":
                return "Up";
            case "Tab":
                return "U+0009";
            case "Escape":
                return "U+001B";
            default:
                return this.key;
        } 
    }
});
```
